pkg/asset/agent/interactive/install_config.go

```diff
@@ -0,0 +1,124 @@
+package interactive
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"sigs.k8s.io/yaml"
+
+	"github.com/openshift/installer/pkg/asset"
+)
+
+var (
+	interactiveInstallConfigFilename = "interactive-install-config.yaml"
+
+	defaultPullSecret = `{"auths":{"":{"auth":"dXNlcjpwYXNz"}}}` //nolint:gosec // no sensitive info
+)
+
+// InstallConfig defines the default setting for the cluster
+// when using the interfactive disconnected workflow.
+type InstallConfig struct {
+	config Config
+}
+
+// Config is used to read the content of the configuration file.
+type Config struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	RendezvousIP string   `json:"rendezvousIP,omitempty"`
+	SSHKey       string   `json:"sshKey,omitempty"`
+	PullSecret   string   `json:"pullSecret,omitempty"`
+	Operators    []string `json:"operators,omitempty"`
```

Comment:
We're not planning to put Operator selection in the above-sea-level UI as I understood it.

Reply:
My understanding is that having them specified at the above-sea-level-UI will allow to have a smaller ISO image (which I understand it will still be a two-digits number :) ). Even if we'll decide to adopt a specific OLM operator bundle, I'm not sure if we'll always have to include _all_ the possible operators. I guess this could be something that could be delayed further from the current implementation

Reply:
We won't be able to build images on the fly with custom operator lists. All images will need to be built in advance.
The best we can plausibly do is two images, one with no operators and one with all of them.
But even that is not a 'list your operators' type input, it's more like 'minimal iso' vs. 'regular iso' (although in this case it'd be 'huge iso' vs. 'huge-r iso') choice.
Selecting the list of operators to install is something you want to do as late as possible. (e.g. you may find after running validations that you don't have the right HW requirements for some, and want to change the list. required storage operators are different for SNO (LSO) vs. other topologies (ODF). &c.).